data-prepper-plugins/mongodb/src/main/java/org/opensearch/dataprepper/plugins/mongo/configuration/CollectionConfig.java

```diff
@@ -9,15 +9,17 @@
 public class CollectionConfig {
     private static final String COLLECTION_SPLITTER = "\\.";
     private static final int DEFAULT_STREAM_BATCH_SIZE = 1000;
+    private static final int DEFAULT_PARTITION_COUNT = 100;
+    private static final int DEFAULT_EXPORT_PARTITION_SIZE = 4000;
     @JsonProperty("collection")
     private @NotNull String collection;
 
-    @JsonProperty("export_config")
-    private ExportConfig exportConfig;
-
     @JsonProperty("export")
     private boolean export;
 
+    @JsonProperty("export_partition_size")
```

Comment:
What does this mean exactly?

Reply:
This is the number of records of chunk size the export partition worker will work on when it executes query to DocDB/Mongo. The export workers run in parallel and they work on independent chunk of data.

Reply:
updated to `export_batch_size`